int16 track_id
geometry_msgs/PoseStamped pose
geometry_msgs/TwistStamped twist